### ISSUE TITLE
Increase test coverage

### ISSUE THREAD

**Issue by danielhollas** [pzarabadip/aiida-orca#64]
The current 55% is very nice, but perhaps we should aim higher before the 1.0 release.

Thanks @pzarabadip for enabling Codecov! :+1:

**Comment by ezpzbz** [comment]
The main reason for this is the cclib parsers. 
I need to go through them and see how we can fix this. Certainly something to keep in mind.

**Comment by danielhollas** [comment]
Good point. Perhaps adding more parser tests with various orca outputs is the way to go, with the new test suite it should be quite easy to add.

**Comment by ezpzbz** [comment]
@danielhollas Sure, we can have tests to cover the parser but I am not keen for this.
As discussed before, having `cclib` parser here is a short-term solution to have this plugin up and running. I am uneasy keeping it this way.
I am thinking of rewriting the parser to be native here and also more robust. There are so many hard-coded logics in the parsers such as `if lines[x:y]` that can easily break the parser if minor changes happen on `ORCA` side.
Also, `orcaparser` is very lengthy makes it hard to read and follow.

**Comment by danielhollas** [comment]
If you want to rewrite the parser, I'd say it's even more important to have the tests, otherwise it's going to be very hard not to break things.
